app/models/comment.rb

```diff
@@ -1,4 +1,13 @@
 class Comment < ApplicationRecord
-  belongs_to :user
   belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  attribute :text, :string
+
+  after_save :update_post_counter
```

Comment:
Great job on implementing the requested changes🤝

- [ ] As requested by the previous reviewer, I think you should use the `after_save` to call the `update_comment_counter` in this `comment` model. Instead of the `update_post_counter` method.